roles/jenkins-worker/tasks/main.yml

```diff
@@ -56,6 +56,11 @@
   tags:
     - docker
 
+- name: Create podman config
```

Comment:
This is great but we'd also need to change the scripts to not install docker and install podman instead, no? :]

Reply:
yeah ... I just added it so I don't lose the config... and then, since we use Fedora, I thought that podman is preinstalled 🙈.

> not install docker and install podman instead

as for this part, I wasn't 100% sure... I was thinking that maybe we could keep both initially and gradually update our builds to switch to podman rather than doing it all at once.